docs/shell_functions.rst

```diff
@@ -11,25 +11,25 @@ Here's a basic example that demonstrates specifying a |ShellFunction|_ that is t
 formatted with a list of values at launch time.
 
 .. code-block:: python
+    :caption: globus_compute_shell_function.py
 
-   from globus_compute_sdk import ShellFunction, Executor
+    from globus_compute_sdk import ShellFunction, Executor
 
-   ep_id = <SPECIFY_ENDPOINT_ID>
-   # The cmd will be formatted with kwargs at invocation time
-   bf = ShellFunction("echo '{message}'")
-   with Executor(endpoint_id=ep_id) as ex:
+    ep_id = "<SPECIFY_ENDPOINT_ID>"
+    # The cmd will be formatted with kwargs at invocation time
+    bf = ShellFunction("echo '{message}'")
+    with Executor(endpoint_id=ep_id) as ex:
+        for msg in ("hello", "hola", "bonjour"):
+            future = ex.submit(bf, message=msg)
+            shell_result = future.result()  # ShellFunctions return ShellResults
+            print(shell_result.stdout)
 
-       for msg in ["hello", "hola", "bonjour"]:
-           future = ex.submit(bf, message=msg)
-           shell_result = future.result()  # ShellFunctions return ShellResults
-           print(shell_result.stdout)
-
-   # Executing the above prints:
-   hello
-
-   hola
-
-   bonjour
+    # Executing the above prints:
```

Comment:
same triple quotes vs. prefixing it all with #'s as above

Reply:
Same resolution as above?